absl/utility/CMakeLists.txt

```diff
@@ -44,7 +44,7 @@ absl_library(
 
 # test utility_test
 set(UTILITY_TEST_SRC "utility_test.cc")
-set(UTILITY_TEST_PUBLIC_LIBRARIES absl::utility)
+set(UTILITY_TEST_PUBLIC_LIBRARIES absl::utility absl::strings)
```

Comment:
and absl::memory I think

Reply:
OK, agree. But it looks this deps is coming transitively though.